.circleci/check_version.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Collect the name and version from `pyproject.toml`
+NAME=$(cat pyproject.toml | grep name | sed 's/"//g' | awk '{print $3}')
+VERSION=$(cat pyproject.toml | grep version | sed 's/"//g' | awk '{print $3}')
+
+# Running `pip install packaganame==` returns this in STDERR
+#   ERROR: Could not find a version that satisfies the requirement packagename== (from versions: 0.1.0, 0.1.1, 0.1.2)
+
+# |& pipes into STDOUT both STDOUT and STDERR
+pip install "$NAME==" |& grep $VERSION
+
```

Comment:
Let me see if I follow correctly the code:
- Extract name and version of the package on the repo
- Try to install `package==version`
- If you can install it means that the version is incorrect?

Not sure if it's my less experience on `bash` or that some of the algorith will need to be explicit either code or commit message.

Also, being this on `pr` what would happen with multiple PRs opened at the same time? 

Why the name is extracted from the project file, are you expected to be change the name of the package?

Reply:
> Try to install package==version

When you run the command `pip install requests==` (notice the lack of a version number), you get a list of all published versions, that `grep` checks if the version we have, is there.

If there is a version in the output, it means it has been published already.

> Not sure if it's my less experience on bash or that some of the algorith will need to be explicit either code or commit message.

I tried to be explicit on the comments, if the explanation above is clearer, I'm happy to change the comment appropriately.

> Also, being this on pr what would happen with multiple PRs opened at the same time?

The same thing that happens now with the other packages, you need to bump each PR.

> Why the name is extracted from the project file, are you expected to be change the name of the package?

The script shouldn't be tied to the package, nor the package name.

Reply:
Can you explain further : 

> The script shouldn't be tied to the package, nor the package name.

Advantages of it for example?